specs/Menu.md

```diff
@@ -1,731 +1,679 @@
 # Menu
 
-## TODOS
+## Background
 
-- Get reviews
-- Write summary at end
-- Checkin to github
-- Write prototype
-- Finalize structure
-- Finalize API
-- Final review
+### Definition
 
-## Menu Basics
+This spec defines the default function of a `Menu` as an interactive component that displays a list of options that can be represented by a range possible states. Possible variants are defined in [the relevant section](#variants)
 
-A menu is a way of displaying items that a user may be able to interact with. Menus vary quite a bit based on implementation and usage. Some are horizontal, some have collapsable sections, and others are just a list. Likewise menu items are equally varied. Some may just convey information but cannot be interacted with like headers or dividers while others are either buttons, links, or expand into a submenu.
+The `Menu` should be displayed on a temporary surface that interrupts the normal flow of content. The temporary surface should be triggered by an external user action such as (but not limited to) a click on a button or other UI control.
 
-The w3 specifications for a menu suggest that it is only for user actions, not user input. Most varieties of menus interpret this as being for navigation.
+The interactions that result in the dismiss/removal of the `Menu` component should be configurable.
 
-Menus are often combined with a popover or similar floating component so the menu appears over other items and is subsequently dismissed.
+## Prior art
 
-### MenuItem
+As a part of the spec definitions in Fluent UI, a research effort has been made through [Open UI](https://open-ui.org/). The current research proposal is available as an open source contribution undergoing review ([research proposal](https://github.com/WICG/open-ui/pull/249))
 
-Most menus have a concept of a specific menuitem. It is an integral part of the menu and will have some of its own sections throughout.
+## Comparison of `@fluentui/react` and `@fluentui/react-northstar`
 
-## References
+- All mentions of v7 or v8 == `@fluentui/react` ([docsite](https://developer.microsoft.com/en-us/fluentui#/))
+- All mentions of v0 == `@fluentui/react-northstar` ([docsite](https://fluentsite.z22.web.core.windows.net/))
 
-[CodeSandbox of implemented versions](https://codesandbox.io/s/menuexamples-uybsr)
+The most relevant comparison that can be achieved between the two libraries is between `ContextualMenu` in v7 and a combination of `Menu`, `Popup` and `ToolbarItem` in v0.
```

Comment:
I would like to have more detailed mapping from converged to v7/8 and v0.
Once the converged `Menu` is implemented, what exactly will it replace in existing libraries?
According to design spec, for v0 it will not replace `Menu` but rather a `MenuButton` and perhaps some other scenarios?

Reply:
The spec just grows larger... :-) I will try to clarify this more

just FYI: I propose a `MenuList` component that can be adequately used to replace v0 `Menu` where as a `MenuButton` should be implemented using `Menu` pretty easily

in v7 the contender here is [ContextualMenu](https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu)